lighthouse-core/config/default.js

```diff
@@ -1,303 +1,321 @@
+    'a11y-element-names': {
+      title: 'Elements Have Discernable Names',
+      description: 'Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content.'
     },
-    "a11y-language": {
-      "title": "Page Specifies Valid Language",
-      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
+    'a11y-language': {
+      title: 'Page Specifies Valid Language',
+      description: 'Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content.'
     },
-    "a11y-meta": {
-      "title": "Meta Tags Used Properly",
-      "description": "Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content."
+    'a11y-meta': {
+      title: 'Meta Tags Used Properly',
+      description: 'Screen readers and other assistive technologies require annotations to understand otherwise ambiguous content.'
     },
-    "manual-pwa-checks": {
-      "title": "Manual checks to verify",
-      "description": "These audits are required by the baseline " +
-          "[PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist) but are " +
-          "not automatically checked by Lighthouse. They do not affect your score but it's important that you verify them manually."
+    'manual-pwa-checks': {
+      title: 'Manual checks to verify',
+      description: 'These audits are required by the baseline ' +
+          '[PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist) but are ' +
+          'not automatically checked by Lighthouse. They do not affect your score but it\'s important that you verify them manually.'
     },
   },
-  "categories": {
-    "pwa": {
-      "name": "Progressive Web App",
-      "weight": 1,
-      "description": "These audits validate the aspects of a Progressive Web App, as specified by the baseline [PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist).",
-      "audits": [
-        {"id": "service-worker", "weight": 1},
-        {"id": "works-offline", "weight": 1},
-        {"id": "without-javascript", "weight": 1},
-        {"id": "is-on-https", "weight": 1},
-        {"id": "redirects-http", "weight": 1},
-        {"id": "load-fast-enough-for-pwa", "weight": 1},
-        {"id": "webapp-install-banner", "weight": 1},
-        {"id": "splash-screen", "weight": 1},
-        {"id": "themed-omnibox", "weight": 1},
-        {"id": "viewport", "weight": 1},
-        {"id": "content-width", "weight": 1},
-        {"id": "pwa-cross-browser", "weight": 0, "group": "manual-pwa-checks"},
-        {"id": "pwa-page-transitions", "weight": 0, "group": "manual-pwa-checks"},
-        {"id": "pwa-each-page-has-url", "weight": 0, "group": "manual-pwa-checks"}
+  categories: {
+    'pwa': {
```

Comment:
> you can unquote these category titles

eslint doesn't like because `best-practices` is on the same level and needs quotes